src/drc/api/viewsets.py

```diff
@@ -95,10 +101,42 @@ class EnkelvoudigInformatieObjectViewSet(NotificationViewSetMixin,
         'destroy': SCOPE_DOCUMENTEN_ALLES_VERWIJDEREN,
         'update': SCOPE_DOCUMENTEN_BIJWERKEN,
         'partial_update': SCOPE_DOCUMENTEN_BIJWERKEN,
+        'lock': SCOPE_DOCUMENTEN_LOCK,
+        'unlock': SCOPE_DOCUMENTEN_LOCK | SCOPE_DOCUMENTEN_GEFORCEERD_UNLOCK
     }
     notifications_kanaal = KANAAL_DOCUMENTEN
     audit = AUDIT_DRC
 
+    @swagger_auto_schema(request_body=LockEnkelvoudigInformatieObjectSerializer,
+                         responses={200: LockEnkelvoudigInformatieObjectSerializer})
+    @action(detail=True, methods=['post'])
+    def lock(self, request, *args, **kwargs):
+        eio = self.get_object()
+        lock_serializer = LockEnkelvoudigInformatieObjectSerializer(eio, data=request.data)
+        lock_serializer.is_valid(raise_exception=True)
+        lock_serializer.save()
+        return Response(lock_serializer.data)
+
+    @swagger_auto_schema(request_body=UnlockEnkelvoudigInformatieObjectSerializer,
+                         responses={200: UnlockEnkelvoudigInformatieObjectSerializer})
```

Comment:
see above for the error response serializers

Reply:
fixed